### ISSUE TITLE
Sequence contains no elements

### ISSUE THREAD

**Issue by paulomorgado** [KirillOsenkov/MSBuildStructuredLog#437]
While opening a binlog from Visual Studio, I got this:

```text
Error while analyzing build. Sorry about that. Please Ctrl+C to copy this text and file an issue on https://github.com/KirillOsenkov/MSBuildStructuredLog/issues/new 

System.InvalidOperationException: Sequence contains no elements

   at System.Linq.Enumerable.Max(IEnumerable`1 source)

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.AnalyzeEvaluation(NamedNode folder) in C:\MSBuildStructuredLog\src\StructuredLogViewer.Core\BuildAnalyzer.cs:line 126

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.ProcessBeforeChildrenVisited(TreeNode node) in C:\MSBuildStructuredLog\src\StructuredLogViewer.Core\BuildAnalyzer.cs:line 121

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node) in C:\MSBuildStructuredLog\src\StructuredLogViewer.Core\BuildAnalyzer.cs:line 77

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node) in C:\MSBuildStructuredLog\src\StructuredLogViewer.Core\BuildAnalyzer.cs:line 79

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Analyze() in C:\MSBuildStructuredLog\src\StructuredLogViewer.Core\BuildAnalyzer.cs:line 63

   at Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.AnalyzeBuild(Build build) in C:\MSBuildStructuredLog\src\StructuredLogViewer.Core\BuildAnalyzer.cs:line 35
```

**Comment by bording** [comment]
I'm also seeing this happen.

**Comment by KirillOsenkov** [comment]
Thanks, fixed in 2.1.303
